assets/js/googlesitekit-data.js

```diff
@@ -1,10 +1,22 @@
 /**
  * External dependencies
  */
-import Data from 'assets/js/googlesitekit/data';
+
+/**
+ * Internal dependencies
+ */
+import SiteKitRegistry from 'assets/js/googlesitekit/data';
+import { collect, collectReducers } from 'assets/js/googlesitekit/data/utils';
 
 if ( typeof global.googlesitekit === 'undefined' ) {
 	global.googlesitekit = {};
 }
 
-global.googlesitekit.data = Data;
+const registry = SiteKitRegistry;
+if ( typeof global.googlesitekit.data === 'undefined' ) {
+	global.googlesitekit.data = { collect, collectReducers, registry };
+}
+
+export { collect, collectReducers, registry };
+
+export default global.googlesitekit.data;
```

Comment:
These two export statements should be replaced with a single `export default Data`, where `Data` would be the object created above and assigned to `global.googlesitekit.data`.